github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-postsubmits.yaml

```
@@ -57,3 +57,33 @@ postsubmits:
           resources:
             requests:
               memory: "8Gi"
+
+    - name: publish-centos8-realtime-container-image
+      always_run: false
```

To fix [this check](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/1515/pull-project-infra-check-testgrid-config/1430170427614826496) you should add the annotation that excludes this job from testgrid

```suggestion
      annotations:
        testgrid-create-test-group: "false"
      always_run: false
```